src/service_daemon.rs

```diff
@@ -1371,23 +1400,32 @@ impl Zeroconf {
                     }
                 }
             } else {
-                if qtype == TYPE_A || qtype == TYPE_ANY {
+                if qtype == TYPE_A || qtype == TYPE_AAAA || qtype == TYPE_ANY {
                     for service in self.my_services.values() {
                         if service.get_hostname() == question.entry.name.to_lowercase() {
                             let intf_addrs = service.get_addrs_on_intf(&intf_sock.intf);
-                            if intf_addrs.is_empty() && qtype == TYPE_A {
+                            if intf_addrs.is_empty() && (qtype == TYPE_A || qtype == TYPE_AAAA) {
+                                let t = match qtype {
+                                    TYPE_A => "TYPE_A",
+                                    TYPE_AAAA => "TYPE_AAAA",
+                                    _ => panic!(),
```

Comment:
To avoid confusion, can you please do one of these two things:

1) use something str like "invalid type" to replace `panic`,

or 

2) add a function like `fn qtype_str(qtype: u16) -> &str` that implements conversion for possible qtype values (including a hard-coded &str for invalid values).  And then use the function here?

Reply:
I went with the first solution,  when/if qtype is an enum is will be easier to implement display on it